client/angular.json

```diff
@@ -108,7 +108,7 @@
           }
         },
         "e2e": {
-          "builder": "@nrwl/cypress:cypress",
+          "builder": "@cypress/schematic:cypress",
           "options": {
```

Comment:
The defaults when you add this schematic are to add these options:
```js
"watch": true,
"headless": false
```
This turns `ng e2e` from something that runs once headless to something that essentially runs `ng serve` and `cypress open` at the same time .

The behavior we are used to from `ng e2e` has been moved to the `ng run client:cypress-run`  from this new architect:
```json
"cypress-run": {
  "builder": "@cypress/schematic:cypress",
  "options": {
    "devServerTarget": "client:serve"
  },
  "configurations": {
    "production": {
      "devServerTarget": "client:serve:production"
    }
  }
},
```
It also by default adds a new `ng run client:cypress-open` command:
```json
"cypress-open": {
  "builder": "@cypress/schematic:cypress",
  "options": {
    "watch": true,
    "headless": false
  }
},
``` 

Though from my testing if we leave it as is everything still works as it did and that's probably fine. `ng e2e` runs the E2E tests once without opening the Cypress GIU (unexpectdly, setting only `"headless": false` in the options without `"watch": true` has no effect) and running `ng e2e --watch` runs the dev server in watch mode and opens the Cypress GUI.

So after all this, I actually don't think we should change anything here. You _could_ add the new commands but they're probably just redundant.